helm/zot/values.schema.json

```diff
@@ -1,412 +1,864 @@
 {
-    "$schema": "http://json-schema.org/schema#",
-    "type": "object",
-    "properties": {
-        "configFiles": {
-            "type": "object",
-            "properties": {
-                "config.json": {
-                    "type": "string"
+  "additionalProperties": false,
+  "properties": {
+    "configFiles": {
+      "additionalProperties": false,
+      "properties": {
+        "config.json": {
+          "default": "{\n  \"storage\": { \"rootDirectory\": \"/var/lib/registry\" },\n  \"http\": { \"address\": \"0.0.0.0\", \"port\": \"5000\" },\n  \"log\": { \"level\": \"debug\" }\n}",
+          "title": "config.json",
+          "type": "string"
+        }
+      },
+      "description": "If mountConfig is true the chart creates the '$CHART_RELEASE-config', if it\ndoes not exist the user is in charge of managing it (as this file includes a\nsample file you have to add it empty to handle it externally) ... note that\nthe service does not reload the configFiles once mounted, so you need to\ndelete the pods to create new ones to use the new values.",
+      "title": "configFiles",
```

Comment:
Having the property name repeated as a title isn't useful.

Reply:
We can disable it by using `-k title` if we don't want it